backend/fms_core/resources/extraction.py

```diff
@@ -106,6 +106,9 @@ def import_field(self, field, obj, data, is_m2m=False):
             # initializing the volume history of the EXTRACTED sample, so the
             # actual history entry is of the "normal" type (UPDATE).
             vol = blank_str_to_none(data.get("Volume (uL)"))  # "" -> None for CSVs
+
+            obj.volume = vol
```

Comment:
i would set obj.volume directly above and use it below to make sure not to leave those intermediate variables floating when we remove the volume_history, but it is not that important ...

Reply:
I dont understand, where would you put obj.volume = vol ?

Reply:
My point was the line is not useful.

Reply:
The blank line?